lib/timeline/component/item/Item.js

```diff
@@ -148,7 +164,7 @@ Item.prototype.repositionY = function() {
 Item.prototype._repaintDeleteButton = function (anchor) {
   var editable = (this.options.editable.remove &&
                   this.options.editable.overrideItems)
-                  || (this.data.editable === true &&
+                  || (this.editable.remove &&
                   !this.options.editable.overrideItems);
```

Comment:
What about: `this.data.editable.remove && !this.options.editable.overrideItems`?